src/Symfony/Component/Notifier/Bridge/Lox24/Lox24Options.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Lox24;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Andrei Lebedev <andrew.lebedev@gmail.com>
+ */
+final class Lox24Options implements MessageOptionsInterface
+{
+    public function __construct(
+        private array $options = [],
+    ) {
+    }
+
+    public function toArray(): array
+    {
+        return $this->options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return null;
+    }
+
+    /**
+     * DateTime object of SMS the delivery time.
+     * If Null or not set, the message will be sent immediately.
+     */
+    public function deliveryAt(?\DateTimeInterface $deliveryAt): self
+    {
+        $this->options['delivery_at'] = $deliveryAt ? $deliveryAt->getTimestamp() : 0;
+
+        return $this;
+    }
+
+    /**
+     * The language of the voice message.
+     * If set 'auto', the automatic language detection by message text will be used.
+     */
+    public function voiceLanguage(VoiceLanguage $language): self
+    {
+        if (VoiceLanguage::Auto === $language) {
+            unset($this->options['voice_lang']);
+        } else {
+            $this->options['voice_lang'] = $language->value;
+        }
+
+        return $this;
+    }
+
+    /**
+     * If True deletes the message from the LOX24 database after delivery.
+     */
+    public function textDelete(bool $isTextDelete): self
```

Comment:
```suggestion
    /**
     * If True deletes the message from the LOX24 database after delivery.
     */
    public function deleteTextAfterSending(bool $deleteText): self
```
this way the method is more self-explanatory

Reply:
fixed

Reply:
I would also rename the used variable to deleteText